gui/src-tauri/tauri.conf.json

```
@@ -78,7 +78,7 @@
         "decorations": false,
         "transparent": false,
         "fileDropEnabled": false,
-        "visible": true
+        "visible": false
```

```suggestion
        "visible": true
```
This needs to be true or else the window is invisible .-.

---

That’s not what visible means, and weird I tested it in my side

---

The window is invisible if false and visible if true tho…

---

Nvm, after further testing, ig that doesn't matter.
This still doesn't fix the window bug for me tho